sulong/runBcFile.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Your own personal scripts and temporary tests should not be committed to the repository. Tests you want to keep should be put in `tests/*`. Also, please never commit compilation output if it is at all avoidable.